src/constants.ts

```diff
@@ -21,7 +21,7 @@ export const GDPR_PURPOSES: [PrivacyRegime[], TrackingPurpose[]] = [
 export const DEFAULT_REGIME_TRACKING_PURPOSE_SCOPES: RegimePurposeScopesConfig =
   [
     GDPR_PURPOSES,
-    [['CPRA', 'CDPA', 'CPA', 'NEVADA_SB220'], ['SaleOfInfo']],
+    [['CPRA', 'CDPA', 'CPA', 'NEVADA_SB220', 'US_DNSS'], ['SaleOfInfo']],
```

Comment:
Should we deprecate the other default US regimes as well since this one will cover all of them?

Reply:
@eligrey i think CX will advise customers to do so! All of these are added to the bundle by default but they can be easily removed